### PR TITLE
Add support for IPv6 to UDPClient

### PR DESCRIPTION
**Pull request by cockroach**
The UDP client does not support IPv6 connections at the moment. This change would fix that.

### CODE REVIEW

File: pythonosc/osc_message_builder.py [hunk 1]
```diff
@@ -21,13 +21,14 @@ class OscMessageBuilder(object):
     ARG_TYPE_MIDI = "m"
     ARG_TYPE_TRUE = "T"
     ARG_TYPE_FALSE = "F"
+    ARG_TYPE_NIL = "N"
```
Comment (attwad) [inline pythonosc/osc_message_builder.py]: you already sent that change in a different PR, please revert

Comment (cockroach) [reply pythonosc/osc_message_builder.py]: Damn, sorry for the mess, I must have had a particularly bad day.

I'll clean this up and try again.
